openapi/paths/v1/billruns/bill_run.yml

```diff
@@ -1,6 +1,10 @@
 get:
   operationId: ViewBillRun
-  description: "Request to view a single (billed or unbilled) bill run based on the bill run ID. Triggers generation of summary stats, which are contained in the response.Request may include optional other parameters which result in response being narrowed down to cover a subset of transactions contained within the bill run. List of individual transactions is not included in the response."
+  description: "Request to view a single (billed or unbilled) bill run based on the bill run ID. Triggers generation of summary stats, which are contained in the response.Request may include optional other parameters which result in response being narrowed down to cover a subset of transactions contained within the bill run. List of individual transactions is not included in the response.
```

Comment:
Do we need more detail here on the summary generation? ie. that it only generates the summary if it's been cleared by adding or removing a transaction, and a holding response may be returned if this takes more than a few seconds?

Reply:
💯  Good point. We didn't have this when I first started documenting the API but now is a good time to capture this.